lib/Cake/Test/Case/Network/Email/CakeEmailTest.php

```diff
@@ -2449,4 +2449,22 @@ public function assertLineLengths($message) {
 		}
 	}
 
+/**
+ * Test if the EmailConfig::$default configuration is read when present
+ *
+ * @return void
+ */
+	public function testDefaultConfig() {
+		$defaultConfig = new File(APP . 'Config' . DS . 'email.php.default');
+		$emailConfig = new File(APP . 'Config' . DS . 'email.php');
+		$configExists = $emailConfig->exists();
+		$this->skipIf(!$defaultConfig->copy(APP . 'Config' . DS . 'email.php', false), 'Unable to copy default config');
+
+		$Email = new CakeEmail();
+		$this->assertEquals('you@localhost', current($Email->from()));
+		if ($configExists) {
+			$emailConfig->delete();
```

Comment:
Hmm, this might actually delete existing config files when run in live apps. Have to think of a safer way to clean up.